package.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "@bcgov-nr/nr-fsa-theme",
-    "version": "1.0.8",
+    "version": "1.0.10",
```

Comment:
Hi Nick, I just realized that we don't need to do version change in package.json, the pipeline will handle the remote version. If we change it to the latest version, it will have conflict with the remote version. Let's not change this

Reply:
Right, thanks! I'll switch it back to 1.0.8.